src/ui/blockUi-editor.js

```diff
@@ -16,12 +16,34 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.
     // the base blockUi for editing individual blocks, contains shared elements
     fluid.defaults("sjrk.storyTelling.blockUi.editor", {
         gradeNames: ["sjrk.storyTelling.blockUi"],
+        members: {
+            // the ID of the story of which the block being edited is a part
+            storyId: ""
+        },
         model: {
+            moveBlockDownDisabled: true,
+            moveBlockUpDisabled: true
+        },
+        modelRelay: {
             // reorder buttons are disabled depending on the block's order:
-            // if the block is first in the order, the "up" button is disabled
             // if the block is last in the order, the "down" button is disabled
-            moveBlockDownDisabled: "{block}.model.lastInOrder",
-            moveBlockUpDisabled: "{block}.model.firstInOrder"
+            blockLastInOrderToDownButtonDisabled: {
+                source: "{block}.model.lastInOrder",
+                target: "moveBlockDownDisabled",
+                singleTransform: {
+                    type: "fluid.transforms.value"
```

Comment:
should this be `fluid.transforms.identity`?

Reply:
These two functions are aliases of one another, and so I chose to use `value` since it's possibly easier to understand at a glance. That said, an Infusion developer would be used to seeing `identity`, and we do use the latter in our tests. Do you have a preference?

Reply:
identity might just be clearer about the expectations of what will happen, which is to say, no model transformation.